docs/articles/dockerfile_best-practices.md

```diff
@@ -153,9 +153,12 @@ maintainers. If you know there’s a particular package, `foo`, that needs to be
 updated, use `apt-get install -y foo` and it will update automatically.
 
 * Do write instructions like:
-
-    RUN apt-get update && apt-get install -y package-bar package-foo package-baz
-
+```
+    RUN apt-get update && apt-get install -y \
+        package-bar \
+        package-foo \
+        package-baz
+```
```

Comment:
Could you use "standard" Markdown here (i.e., just indentation, not the triple back-ticks)?

The tooling used for the documentation currently doesn't properly support GitHub-flavoured Markdown (it's on our list, but not there yet)